src/applications/pact-act/utilities/page-setup.js

```diff
@@ -35,8 +35,8 @@ export const applyFocus = (parentId, headerHasFocused, setHeaderHasFocused) => {
     }, 500);
   }
 };
+
 export const pageSetup = () => {
   window.scrollTo({ top: 0, left: 0, behavior: 'instant' });
   waitForRenderThenFocus('h1');
-  document.title = 'Learn how the PACT Act may affect you | Veterans Affairs';
```

Comment:
Moved the title to the top of the application (`PactActApp`) as it is no longer dynamically generated (and doesn't need to be called in each question's `pageSetup` function)